common/common.c

```diff
@@ -54,6 +54,7 @@ void PRINT_WRAPPER( print_t priority, const char* format, ... ) {
 
 	if ( NULL != o ) {
 		vfprintf( o, format, args );
+		fflush( o );
```

Comment:
Avoid delaying the buffered stdout / stderr streams by flushing immediately. Otherwise journalctl takes forever to show changes. kb 4275